Automattic/wp-calypso
# Automattic for Agencies: Implement license details view with mock data

Resolves https://github.com/Automattic/jetpack-genesis/issues/252
Resolves https://github.com/Automattic/jetpack-genesis/issues/257

## Proposed Changes

This PR implements:

- Details view for the License
- Mock API data for license bundles

## Testing Instructions

NOTE: Please note the license data is mocked in the UI currently. It will be loaded only in the UI until the feature flag is enabled on other ENVs. The actions on the details view will be implemented on another PR. Ignore the button styles, it will be fixed via a different PR later.

- Switch to the branch `git checkout add/a4a-license-list-expanded-view`.
- Start the server by running `yarn start-a8c-for-agencies`.
- Click the Purchases menu item > Verify that you can see the Licenses page with 2 licenses. Expand the first one and verify that the detailed view for the bundle is loaded with a list of bundles. 

<img width="1425" alt="Screenshot 2024-02-28 at 3 27 16 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/0d9fa4eb-90b0-445d-88a4-9022d9343d80">

<img width="1425" alt="Screenshot 2024-02-28 at 3 27 32 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/3d13625c-0cde-4149-83ac-f37af907d343">

- Open the 2nd license and verify the details view is displayed as shown below:

<img width="1425" alt="Screenshot 2024-02-28 at 3 27 37 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/5966a309-839a-45ff-9ffd-407dbf4e6b9e">


## Pre-merge Checklist
